### DISCUSSION TITLE
Provide Better Contrast for Selected Row

### DISCUSSION THREAD

**Discussion by atomdmac** [ankitpokhrel/jira-cli#379] [Ideas] [answered]
Currently, the selected row in the list view is indicated by a slightly-darker grey.  This can be very difficult to see in some lighting conditions.  For example, if I do `jira sprints --current`, the resulting list is affected.

It would be nice if either:
* The contrast could be increased
* A more prominent visual indicator could be added (such as an arrow or dot appearing next to the row)

**Comment by ankitpokhrel** [comment, marked as answer]
Hi @atomdmac, you can now customize the UI of the selected row (#419). List of valid colors can be found [here](https://github.com/gdamore/tcell/blob/e15e96cbcae1cd93791c337294f2cb37838b8788/color.go#L846-L991). The config needs to be added manually for now and will be overwritten if you run `jira init` again.

```yaml
tui:
  selection:
    foreground: red
    background: navy
    bold: true
```

The changes are available in the main branch. For now you either need to [compile it locally](https://github.com/ankitpokhrel/jira-cli#development) or use following methods to install. Changes will be available in the new release possibly in a few weeks.

With [Golang](https://go.dev/)

```sh
go install github.com/ankitpokhrel/jira-cli/cmd/jira@main
```

With [docker](https://www.docker.com/)

```sh
docker pull ghcr.io/ankitpokhrel/jira-cli:main
```
